### DISCUSSION TITLE
Panopticon - Edit and Add tasks

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#592] [Q&A (Community Support)] [answered]
I started using Panopticon a few months ago. I set up  Daily Backup. It all works fine, but I want to change the backup time. I can't see a way to edit the task or create a new task? On the Task panel I can enable or disable the task, but there is no option to edit it? Am I being stupid here and missing something? See screenshot
![Akeeba-Panopticon (1)](https://github.com/akeeba/panopticon/assets/95095238/1de8e4da-43f3-4225-828a-f0c5dd40bf8b)

**Comment by nikosdion** [comment, marked as answer]
Main page, click on the site.

Scroll down to the "Backup using Akeeba Backup Professional" section and expand it.

Click on Schedule Backups.

You can now edit the scheduled backup tasks for this specific site.

**Reply by david-unwin** [reply]
Thank you....much appreciated.

**Reply by nikosdion** [reply]
You're welcome!
